pallets/marketplace/src/types.rs

```diff
@@ -26,20 +26,15 @@ pub type JobRegistrationForMarketplace<T> =
 
 /// Struct defining the extra fields for a `JobRegistration`.
 #[derive(RuntimeDebug, Encode, Decode, TypeInfo, Clone, PartialEq, Eq)]
-pub struct RegistrationExtra<Reward, Balance, AccountId>
-where
-    Reward: Parameter + Member,
-{
-    pub requirements: JobRequirements<Reward, AccountId>,
+pub struct RegistrationExtra<Balance, AccountId> {
+    pub requirements: JobRequirements<Balance, AccountId>,
```

Comment:
you merged the stale Reward type with the Balance type, however there are not the same. Balance used to be the native token of the target chain and Reward the multiasset (now refactored to always be an ACRST amount, probably an u128). In JobRequirements we still call the generic parameter `Reward`.

Reply:
The reward field now uses the same type as `Balances = u128`. The `expected_fulfillment_fee` could field could be ignored since it is only used in the target chain. https://github.com/Acurast/acurast-core/pull/94/files#diff-45ba4510baf2604ab580b48f00d8d7233eebbe99cd4bc682d56d3fc74dead965R271

I can remove it from the structure in this PR. Let me know what you think

Reply:
ok, so if we remove it now or later, I still would name it consistently Balance or Reward (I prefer Reward).

Reply:
renamed